py/torch_tensorrt/dynamo/backend/lowering/module_substitutions/maxpool1d.py

```diff
@@ -0,0 +1,75 @@
+from typing import Dict, Tuple
```

Comment:
It would be cool to have a sphinx tutorial on how to do this from an external user perspective. could be as easy as removing maxpool1d from the registry then walking through all the parts.

Reply:
Agreed. I will add this, in addition to documentation on how to ensure all the relevant code is registered (`__init__.py` imports)

Reply:
To avoid clashing with or making dependencies on #1966 and #1967, I'll add more commenting to this file and then port it over to a formatted `.py` file once those PR are merged.